packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -614,3 +615,64 @@ export interface GetIndexerMakerStatisticsResponse {
   rewardCoefficient: BigDecimal;
   makers: IndexerMaker[];
 }
+
+export interface GetIndexerLeaderboardParams {
+  contestId: number;
+  rankType: IndexerLeaderboardRankType;
+  // Min rank inclusive
+  startCursor?: string;
+  limit?: number;
+}
+
+export interface IndexerLeaderboardParticipant {
+  subaccount: Subaccount;
+  contestId: number;
+  pnl: BigDecimal;
+  pnlRank: BigDecimal;
+  percentRoi: BigDecimal;
+  roiRank: BigDecimal;
+  // Float indicating the ending account value at the time the snapshot was taken i.e: at updateTime
+  accountValue: BigDecimal;
+  // Seconds
+  updateTime: BigDecimal;
+}
+
+export interface GetIndexerLeaderboardResponse {
+  participants: IndexerLeaderboardParticipant[];
```

Comment:
Hey @yruej301, I'm currently working on the frontend piece of this. 

Should there be some kind of `meta.nextCursor` or `nextPageParam` returned here as well to help with pagination?

Reply:
Are we able to derive this with the query params of startCursor & limit?

Reply:
Actually, we typically have additional processing to support pagination in `IndexerClient` (i.e. adding `WithPaginationMeta`). @yruej301 are you able to add this for this query as well?

Reply:
yup, i will add